backend/capellacollab/core/database/__init__.py

```diff
@@ -23,7 +24,7 @@ class Base(orm.DeclarativeBase):
 from . import models  # isort:skip # pylint: disable=unused-import
 
 
-def get_db() -> orm.Session:
+def get_db() -> t.Generator[orm.Session, None, None]:
```

Comment:
Could also use `t.Iterator` for generators that you cannot send anything to:

```suggestion
def get_db() -> t.Iterator[orm.Session]:
```

https://docs.python.org/3/library/collections.abc.html#collections-abstract-base-classes